lib/ukschedcoop/schedcoop.c

```diff
@@ -115,6 +117,7 @@ static void schedcoop_schedule(struct uk_sched *s)
 		next = &c->idle;
 	}
 
+	uk_thread_set_queueable(prev);
```

Comment:
This can set the queueable flag for `next` if `next` == `prev` (see line 109 in the updated file). Not sure if that's intentional.